specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2020-04-01-preview/resourcechanges.json

```diff
@@ -0,0 +1,413 @@
+        },
+        "timestamp": {
+          "description": "The time when the snapshot was created.\nThe snapshot timestamp provides an approximation as to when a modification to a resource was detected.  There can be a difference between the actual modification time and the detection time.  This is due to differences in how operations that modify a resource are processed, versus how operation that record resource snapshots are processed.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "content": {
+          "description": "The resource snapshot content (in resourceChangeDetails response only).",
+          "type": "object"
+        }
+      },
+      "required": [
+        "timestamp"
+      ]
+    },
+    "ResourceChangeDetailsRequestParameters": {
+      "description": "The parameters for a specific change details request.",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Specifies the resource for a change details request.",
+          "type": "string"
+        },
+        "changeId": {
+          "description": "Specifies the change ID.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "resourceId",
+        "changeId"
+      ]
+    },
+    "DateTimeInterval": {
+      "description": "An interval in time specifying the date and time for the inclusive start and exclusive end, i.e. `[start, end)`.",
+      "type": "object",
+      "properties": {
+        "start": {
+          "description": "A datetime indicating the inclusive/closed start of the time interval, i.e. `[`**`start`**`, end)`. Specifying a `start` that occurs chronologically after `end` will result in an error.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "end": {
+          "description": "A datetime indicating the exclusive/open end of the time interval, i.e. `[start, `**`end`**`)`. Specifying an `end` that occurs chronologically before `start` will result in an error.",
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": [
+        "start",
+        "end"
+      ]
+    },
+    "ErrorResponse": {
+      "properties": {
+        "error": {
+          "type": "object",
+          "description": "The error object.",
+          "properties": {
+            "code": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The error code."
+            },
+            "message": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The error message."
+            },
+            "target": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The error target."
+            },
+            "details": {
+              "readOnly": true,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ErrorResponse"
```

Comment:
by any chance should this be #/definitions/ErrorDetails?

Reply:
Good catch! Somehow when I messed it up when moving the API to a new file..